cmd/client/lib/cli.go

```diff
@@ -159,36 +163,39 @@ func Create(c *cli.Context, withInstrumentation bool, opts ...client.Option) (cl
 	return client.Wrap(clients, opts...)
 }
 
-func buildGrpcClient(c *cli.Context, info **chain.Info) ([]client.Client, error) {
-	if c.IsSet(GRPCConnectFlag.Name) {
-		hash := make([]byte, 0)
+func buildGrpcClient(c *cli.Context, info *chain.Info) ([]client.Client, *chain.Info, error) {
+	if !c.IsSet(GRPCConnectFlag.Name) {
+		return nil, info, nil
+	}
 
-		if c.IsSet(HashFlag.Name) {
-			var err error
+	hash := make([]byte, 0)
```

Comment:
is there a practical difference between using `make` vs `var hash []byte` here?

Reply:
Given how we use it, no. I'll address this.